src/main/java/mods/railcraft/common/carts/EntityCartEnergyMFE.java

```diff
@@ -58,7 +59,8 @@ public int getTransferLimit() {
     }
 
     @Override
-    public ItemStack getIC2Item() {
-        return IC2Plugin.getItem("mfeUnit");
+    public IBlockState getDefaultDisplayTile() {
+        return IC2Plugin.getBlockState("te","mfe");
```

Comment:
You have issues for comma spacing. Put a space after the comma because the comma is not in the quotes.